web/annotation_layer_builder.css

```diff
@@ -210,7 +210,7 @@
 
 .annotationLayer .popup {
   position: absolute;
-  z-index: 200;
+  z-index: 2000;
```

Comment:
*Another thing that just occurred to me, although I've not tested it:*
Do you actually need this property, now that an explicit z-index is assigned in the JS code?

Reply:
It seems useless: I removed it.
I changed:
https://github.com/mozilla/pdf.js/pull/15237/files#diff-d7b040092cf6be09a276b785ba03c46e79bc8342cf4ae4fb1bc139dea8513425R1906

because the `+=` was just concatenating "1000" to the zindex string when the `-=` below was subtracting 1000.